### PR TITLE
CP 88 Storage Device Capacity

### PR DESCRIPTION
**Pull request by kchason**
- Add the `observable:storageCapacityInBytes` to the DeviceFacet
- Remove explicit property from `MobileDeviceFacet` and `SIMCardFacet`
- Change the parent class from Facet to DeviceFacet to inherit the appropriate properties
- Make the description for `observable:storageCapacityInBytes` to be more general

### CODE REVIEW

File: ontology/observable/observable.ttl [hunk 1]
```diff
@@ -3724,7 +3730,7 @@ observable:MobileDeviceFacet
 		owl:Class ,
 		sh:NodeShape
 		;
-	rdfs:subClassOf core:Facet ;
+	rdfs:subClassOf observable:DeviceFacet ;
```
Comment (ajnelson-nist) [inline ontology/observable/observable.ttl:3733]: NACK here.  There is a pending design document from @sbarnum to explain why this is a subclass of `core:Facet`.  See [OC-97](https://unifiedcyberontology.atlassian.net/browse/OC-97) for reference.

File: ontology/observable/observable.ttl [hunk 2]
```diff
@@ -5150,7 +5150,7 @@ observable:SIMCardFacet
...
```
Comment (ajnelson-nist) [inline ontology/observable/observable.ttl:5153]: Same issue related to OC-97 here.

**Review by ajnelson-nist — CHANGES_REQUESTED** [review]

File: ontology/observable/observable.ttl (hunk 1 above)
Comment (sbarnum) [reply ontology/observable/observable.ttl:3733]: Agree. This should be subClassOf core:Facet

**Comment by ajnelson-nist** [comment]
@kchason , it seems I can't edit this branch.  Can you please do a catch-up merge to resolve the new conflict, and then apply what @sbarnum and I suggested?

**Pull request closed without merging**